posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,106 @@
+
+The Epiverse-TRACE team is developing a set of tools to help provide analytics support to decision-makers during this outbreak. This post provides an overview of aspects of analytics that could support decision-making in the context of the ongoing outbreak of mpox.
+
+## Common outbreak analytics tasks
+
+Outbreak analytics in the context of the ongoing mpox outbreak involves several tasks that can be handled by existing and emerging R tools. Some of the tasks include estimating the transmission potential, forecasting infection dynamics, estimating severity, and assessing the impact of interventions.
+
+Here, we briefly describe some common tasks, data required, and the ready R tools/packages developed by the Epiverse-TRACE team and the wider community.
+
+### Cleaning and validating data
+
+::: {.callout .callout-info}
+Data cleaning is often the first task in outbreak analytics. This usually involves identifying and correcting errors in the data, standardizing the format of key variables, and ensuring that the data is in a format that is fit for analysis. Data validation is also important to ensure that the data is accurate.
+:::
+
+[`{cleanepi}`](https://epiverse-trace.github.io/cleanepi/)} is useful for cleaning individual-level datasets, and [`{listlist}`](https://epiverse-trace.github.io/linelist/)} can be used to tag and validate key variables in datasets that might change over time. The [`{numberize}`](https://epiverse-trace.github.io/numberize/) package can also be used to convert numbers written as text. It currently has functionality for English, Spanish, and French.
+
+### Estimating transmission potential
+
+::: {.callout .callout-info}
+A key initial question during emerging outbreaks is the transmission potential of the disease. This is typically quantified using parameters such as: the basic reproduction number, $R0$; the time varying reproduction number, $Rt$; and $k$, which captures individual heterogeneity in transmission (i.e. "superspreading" potential). These quantities are useful to assess the potential for further spread of the disease and the impact of interventions.
+:::
+
+#### Population-level transmissibility ($R0$ and $Rt$)
+
+The basic reproduction number, $R0$, is the average number of secondary cases produced by a single infected individual in a completely susceptible population. The instantaneous reproduction number, $Rt$, on the other hand, is the average number of secondary cases produced by a single infected individual at time $t$ in a partially susceptible population. The instantaneous reproduction number is a more useful quantity during an outbreak as it accounts for the impact of interventions and changes in population immunity.
```

Comment:
Above $R_t$ is called the time varying reproduction number - suggest edit for consistency.